dcpy/connectors/edm/publishing.py

```diff
@@ -97,3 +99,26 @@ def upload(
         if latest:
             ## much faster than uploading again
             s3.copy_file(BUCKET, str(key), str(prefix / "latest" / output.name), acl)
+
+
+def read_csv(dataset, version, filepath, **kwargs):
```

Comment:
I'm calling this middle arg "version" but it's blurring some lines - really, right now it's a path to an output folder. Longer term, we need to agree a bit about standardizing how we're outputting data and what a "version" vs a "build" is (dates vs branches etc) but for now this seemed relatively fine

Reply:
I like the choice of "version", it's hard to make a function & it's args accurate for all products before we standardize folder structure